dts/bindings/sensor/ti,ina23x-common.yaml

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2021 Grinn
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+include: i2c-device.yaml
+
+properties:
+    config:
+      type: int
+      required: true
+      description: Configuration register
```

Comment:
Does this correspond to a hardware register? Please document more clearly and consider setting a default value

Reply:
I've updated the description but I didn't set the default value since this register is different for 230/231 and 237 variants.

Reply:
Ok, then please mention this in this the description.

Reply:
Done.